docs/python/machine_learning/synthethic_data/1_synthetic_data.md

```diff
@@ -0,0 +1,177 @@
+# Introduction
+
+In this section we'll explore the generation of synthetic data using SDV library. The SDV offers multiple machine learning models ranging from classical statistical methods (Copulas) to deep learning methods (GANs). Synthetic Data is very important for a number of reasons:
+- Software Testing
+- Access Expansion
+- Pilot New Products
+- Augmented Data
+- Plan scenarios
+
+## Installation
+
+To install we should use Poetry and Pyenv. The version of python should be >= 3.10 and < 3.11. For the analysis done on this library we used Python 3.10.9.
+```
+pyenv install 3.10.9
+curl -sSL https://install.python-poetry.org | python3 -
+
+# In this case we already had a project directory
+poetry init
+poetry env use ~/.pyenv/versions/3.10.9/bin/python
+```
```

Comment:
This should be replaced by a subsection called requirements, where it is mentioned that a the time this was written, de required Python version was >= 3.10 and < 3.11.